web/js/components/animation-widget/gif-post-creation.js

```diff
@@ -71,35 +70,50 @@ export class GifResults extends Component {
                 <div>
                   <b> Size: </b>
                 </div>
-                <div>{size} MB</div>
+                <div>
+                  {size}
+                  {' '}
```

Comment:
This change seems like it might be unwanted extra work.

Reply:
I don't know that I love having the var, the space, and the text on three separate lines.  That said, the linter auto-formats this way if you enter new code or modify existing so, it shouldn't require any work on our part to comply.